nucypher/blockchain/eth/actors.py

```diff
@@ -1826,11 +1827,15 @@ def verify_bidding_correctness(self, gas_limit: int) -> dict:
         if whales:
             raise self.WhaleError(f"Some bidders have bids that are too high: {whales}")
 
+        self.log.debug(f"Starting bidding verification. Next bidder to check: {self.worklock_agent.next_bidder_to_check}")
+
         receipts = dict()
         iteration = 1
         while not self.worklock_agent.bidders_checked():
+            self.transacting_power.activate()  # Refresh TransactingPower
```

Comment:
How is another `TransactingPower` being activated between here and the line above ?(<https://github.com/nucypher/nucypher/pull/1784/files#diff-5c5b901e0ca55ebb2746869c1bc5d209R1665>)